spyderlib/spyder.py

```diff
@@ -485,7 +485,7 @@ def __init__(self, options=None):
         self.is_starting_up = True
         self.is_setting_up = True
 
-        self.dockwidgets_locked = False
+        self.dockwidgets_locked = CONF.get('main', 'panes_locked', True)

```

Comment:
Maybe the argument should be removed from the fuction then, to avoid future similar problems ?